### ISSUE TITLE
Starting block won't allow/show more than one quest

### ISSUE THREAD

**Issue by SmrdutyOrangutan** [PikaMug/Quests#1626]
**What is your Quests version / build number (do _not_ say "latest")?**

4.0.0-rc.4-b191

**Describe the problem and how to replicate it. Any console errors to post _using a pastebin_?**

I'm not sure if it's a bug or intended behavior, I asked on Discord and was redirected here. I also searched for similar problem here and couldn't find anything. To the problem:
If I assign more than one quest to a starting block, the block will show me only the first quest. If I chose different starting block for the second one everything works fine. If I switch the order, problem persists. If I chose starting NPC instead and give him both quests, I can chose from the two.
It doesn't matter what block do I chose, I tried over 10 combinations and none of them worked (full blocks, half blocks, walls, sign...). I guess I just need a confirmation if a block can hold more than one quest.

**If applicable, can you provide an example quest from quests.yml _using a pastebin_?**

**Comment by PikaMug** [comment]
Thanks for the report. Have you checked if it would show you a second quest once you _accept_ the first one? It's likely not designed to show you all quests at once.

**Comment by SmrdutyOrangutan** [comment]
I tested it again. 
Quests #1 and #2 are assigned to a dirt block. I can see only quest #1 and I can accept only #1. When I click the block after #1 is finished, I'll get an announcement, that this quest is already finished. If I put a cooldown on the quest, it will let me know when I can repeat the quest. If I remove #1 from the block, I can see #2 again.

I've created an NPC and assigned both quests to this NPC, after clicking, I can chose which quest to accept.

I've tested it with many, many quests, not just these two, none of them worked when I chose block start, all of them worked when I chose NPC start. It really seems like blocks can hold only one quest at a time

**Comment by SmrdutyOrangutan** [comment]
@PikaMug 

Updated to 4.0.0b218, problem persists with quests created before and after update.

**Comment by PikaMug** [comment]
I was able to replicate this and the issue is two-fold. Only the prompt for accepting Quests _from an NPC_ supports multiple quests - any other scenario has a different prompt which only handles a single quest. Without a rewrite of the latter, multiple block starts on the same block aren't going to be inherently supported.

The second issue is that the plugin checks for completed Quests first, and quits looking if one is found. So once you see the "You have already completed <quest>" message, it stops the search. I'll change this so it continues, but you'll still get the message for _each_ completed quest until the search is completed, _then_ it will offer you the quest.

Tl;dr: An NPC would be preferable if you have many quests in this one location.

**Comment by PikaMug** [comment]
https://ci.codemc.io/job/PikaMug/job/Quests/223/
